_pages/en/it-picture-long-term.md

```diff
@@ -6,12 +6,55 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+### Government IoT Integration 
+
+ESDC does not develop products or services that compete with the private sector. IoT devices will begin to permeate all aspects of life and ESDC should integrate itself into the ecosystem. 
+
+An example of the repercussions of such an implementation can be found [here](https://github.com/sara-sabr/ITStrategy/blob/master/wip-tec/en/human-development-life-cycle.md).
+
+In a service based environment, success is not measured by the number of users your service attracts, but rather the number of users amassed by the services that leverage your service. 
```

Comment:
As in, for example, how many requests from 3rd party applications calling our public APIs?

Reply:
Yeah, this one is a little tricky to measure. For example, if I stand up and end point and the app using my data starts caching information ... 
1. Are they allowed to do that?
2. If they are using data from the service, but no longer making calls to our endpoints (because they cached the data), is that a good thing or not? If we don't care, as long as citizens are using the data, then something like 'number of active accounts using our APIs' would be a better metric ...